addon/config/ember-try.js

```diff
@@ -5,7 +5,7 @@ const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
 
 module.exports = async function () {
   return {
-    useYarn: true,
+    usePnpm: true,
     scenarios: [
       {
         name: 'ember-lts-3.28',
```

Comment:
Do we need this line here?

Reply:
we do not, but it's left over from previous, I believe, if it's ok, I'd like remove that in the next PR: https://github.com/emberjs/ember-qunit/pull/1062/files#diff-e4902c163521c37692a4266ecd9a8321c37b194fb0214e5438dc9c3bce045404  to keep potential for conflicts to a minimum (since this file is changing locations in that PR)